apps/video/locales/video.en-US.properties

```diff
@@ -79,3 +79,25 @@ date-group-header = %B %Y
 share     = Share
 more-info = More info
 delete    = Delete
+
+# The following strings are spoken by screen readers and not shown on the
+# screen.
+video-button.ariaLabel = Video
+select-button.ariaLabel = Select
+delete-button.ariaLabel = Delete
+share-button.ariaLabel = Share
+info-view.ariaLabel = Info
+options-view.ariaLabel = Options
+confirm-dialog.ariaLabel = Confirm
+videos-list.ariaLabel = Videos
+video-toolbar.ariaLabel = Controls
```

Comment:
I don't see where this is used. Should videoToolBar have a l10n-id of 'video-toolbar'?